### PR TITLE
Try to terse up README and make it more concrete

### PR DESCRIPTION
**Pull request by zspencer**
In [our Q1 2022
Retro](https://github.com/zinc-collective/convene/discussions/737), we
discussed not being clear on the Product, and how that gets in the way
of figuring out what to do in our ensemble sessions.

We took an edit pass on the README, making it more clear that it's an
operating system; and trying to integrate Zee's weird hand wavey
ramble-rants into something that people other than Zee could understand.

Co-authored-by: Kelly Hong <KellyAH@users.noreply.github.com>
Co-authored-by: Neer Malathapa <nirmalathapa@users.noreply.github.com>
Co-authored-by: Naomi Quinones <naomiquinones@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,29 +19,22 @@ Convene secure, affordable, always-available digital Spaces where anyone can
 
 ## Overview
 
-Convene serves as the _Means of Connection_. Create a Space to co-conspire,
-relax, learn, play, work, swap recipes, read books, or really anything!
+Convene is an _Operating System_ for **Community Owned Digital Infrastructure**. Once installed
+to the cloud, you access Convene via a web browser or on-device apps.
```
Comment (anaulin) [inline README.md:23]: s/Once installed to cloud/Once you start the server/

If the audience are computer-y people, it seems like they would understand what "starting the server" means.

File: README.md [hunk 2]
```diff
@@ -20,29 +19,22 @@ Convene secure, affordable, always-available digital Spaces where anyone can
...
 
-Convene provides the underlying infrastructure for a _Neighborhood of Spaces_.
-_Operators_ help _Clients_ design their _Spaces_ and keep the Neighborhood up
-and running. _Maintainers_ and _Contributors_ enhance Convene with:
+Convene serves as a community-owned, more ethical alternative to investor-owned online tools like
+SquareSpace, Etsy, Patreon, Zoom, Yelp, and more.
```
Comment (anaulin) [inline README.md:26]: 👍🏼 this whole paragraph

File: README.md [hunk 3]
```diff
@@ -20,29 +19,22 @@ Convene secure, affordable, always-available digital Spaces where anyone can
...
 
-- _Furniture_, fun toys and powerful tools
-- _Utilities_, integrating external services like Stripe, Calendly, Jitsi, and
-  more.
+A Convene Deployment is a _Neighborhood_, which People can create _Spaces_ in, and fill those _Spaces_
```
Comment (anaulin) [inline README.md:28]: Remove capitals from "People", and maybe drop the "can", i.e:

> A Convene Deployment is a _Neighbordhood_, which people can create _Spaces_ in, and fill...

Also, potentially add a period in there to make the second part less awkward (without the dangling preposition), like:
> A Convene Deployment is _Neighborhood_. People create _Spaces_ in a _Neighborhood_, and fill those _Spaces_...

File: README.md [hunk 4]
```diff
@@ -143,7 +138,7 @@ To run `pry` or `byebug`, run `bin/connect web`.
 
 ### Developing Mailers
 
-Set [SMTP_* environment variables to configure Action Mailer](.env.example).
+Set [SMTP\_\* environment variables to configure Action Mailer](.env.example).
```
Comment (anaulin) [inline README.md:141]: Thank you, 🤖

**Review by anaulin — APPROVED** [review]
Thanks for doing this, nicely done and very helpful! 🙌🏼

Some minor suggestions for improvement before merge below.

File: README.md (hunk 1 above)
Comment (KellyAH) [reply README.md:23]: fixed in https://github.com/zinc-collective/convene/pull/749

File: README.md (hunk 3 above)
Comment (KellyAH) [reply README.md:28]: fixed in https://github.com/zinc-collective/convene/pull/749

**Pull request merged**